wiki/HistoricalTimeline.md

```diff
@@ -52,59 +60,69 @@ This timeline is provided mainly to assist with story writing, to make sure that
 
 * 2660: In the Core, which is the end point of one of the major caravan routes, the Syndicate is beginning to gain power as it merges with smaller corporations in the area.
 
+* 2670: A worker's revolution on Nimbus is brutally suppressed by the new Syndicate and their police forces. Allegations of war crimes are made by the Deep, but little is done to punish the growing megacorporation.
+
 * 2680: Because none of the small Rim and Core corporations are as powerful as them, commerce is vastly unbalanced in favor of the Deep. Some of the small corporate governments try to establish their own trade zone, essentially boycotting the Deep and their caravan fleets.
 
 ***
 
-* 2700: A new era of peace and optimism.
+* 2700: A new era of peace and optimism begins.
 
-* 2720: Several independent planets in the Core and Rim begin discussing joining together and forming a representative government with Earth as its capital.
+* 2720: Several independent planets in the Core and Rim begin discussing joining together and forming a representative government with Earth as its capital. 
 
 * 2730: The Republic is formed, initially comprising Earth, the Rim systems closest to Quarg space, and several Core systems close to Earth, plus the few Dirt Belt worlds that are already inhabited.
 
 * 2740: The Republic imposes trade embargoes on the Deep, the Syndicate, and the paradise worlds, exerting pressure on them to join the Republic.
 
 * 2750: After years of nonviolent protests by their citizens, the Syndicate worlds agree to join the Republic, with the local governments becoming democratic instead of being solely administered by the Syndicate itself. Many of the newly elected officials, however, have ties to the Syndicate.
 
-* 2760: The paradise worlds join the Republic, but with concessions that include a restructured Parliament where very populous worlds have greater representation, which gives the paradise planets considerable power.
+* 2760: The paradise worlds join the Republic, but with concessions that include a restructured Parliament where very populous worlds have greater representation, which gives the paradise planets considerable power. Many in sparsely populated systems are outraged.
 
-* 2770: A new wave of colonization is beginning, in part because the terraforming tools of the paradise worlds have led people to believe that any planet can be made perfect for human habitation given enough time.
+* 2770: A new wave of colonization is beginning, in part because the terraforming tools of the paradise worlds have led people to believe that any planet can be made perfect for human habitation given enough time. Permanent colonies are established on mining and farming worlds that before had been inhabited only by temporary worker settlements. 
```

Comment:
```suggestion
* 2770: A new wave of colonization is beginning, in part because the terraforming tools of the Paradise worlds have led people to believe that any planet can be made perfect for human habitation given enough time. Permanent colonies are established on mining and farming worlds that before had been inhabited only by temporary worker settlements. 
```